packages/toolkit/src/view/model/view-model/ModelRuns.tsx

```diff
@@ -0,0 +1,18 @@
+import type { Model } from "instill-sdk";
+import { useParams } from "next/navigation";
+
+import { ModelRun, ModelRunList /* , ModelRun */ } from "./ModelRunViews";
```

Comment:
@orangecoloured Let's reduce this kind of comment in the codebase. Since this didn't have the guidance nor the record purpose